client/my-sites/subscribers/main.tsx

```diff
@@ -1,19 +1,306 @@
 import config from '@automattic/calypso-config';
+import { useLocale } from '@automattic/i18n-utils';
 import { translate } from 'i18n-calypso';
+import moment from 'moment';
+import { Item } from 'calypso/components/breadcrumb';
 import DocumentHead from 'calypso/components/data/document-head';
-import Main from 'calypso/components/main';
+import FixedNavigationHeader from 'calypso/components/fixed-navigation-header';
+import FormCheckbox from 'calypso/components/forms/form-checkbox';
+import './styles.scss';
+import { SubscriberProfile } from './subscriber-profile';
+
+const subscribers = [
```

Comment:
Hi, Paulo!
The good news is I'm removing the mock as the endpoint and the query is already in the trunk. I'll push this change today.

Reply:
Maybe in this case it could be in a different PR/task as you already have the mock implemented. (But of course if you already applied the query you can ignore my message 😄)